docs/source/tutorials/automated-publication-with-ci.md

```diff
@@ -0,0 +1,143 @@
+<!--
+SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR), Forschungszentrum Jülich GmbH
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+<!-- 
+SPDX-FileContributor: Oliver Bertuch
+SPDX-FileContributor: Michael Meinel
+SPDX-FileContributor: Stephan Druskat
+-->
+
+# Set up automatic software publishing
+ 
+```{note}
+This tutorial works for repositories hosted on GitHub, and shows how to automatically publish to Zenodo Sandbox.
+Zenodo Sandbox is a "toy" repository that can be used to try things out.
+
+This tutorial should also work with the "real" Zenodo.
+```
+ 
+## Configure your .gitignore
+ 
+The HERMES workflow (`hermes`) uses temporary caches in `.hermes/`.
+Ignore this directory in your git repository.
+
+Add `.hermes/` to your `.gitignore` file:
+ 
+```bash
+# .gitignore
+
+.hermes/
+```
+
+## Provide additional metadata using CITATION.cff
+ 
+To provide high-quality citation metadata for your project and your publication,
+provide a `CITATION.cff` file in the [Citation File Format](https://citation-file-format.github.io/).
+
+If you don't have one yet,
+use the [cffinit](https://citation-file-format.github.io/cff-initializer-javascript/) website
+to create a `CITATION.cff` file.
+Save it to the root directory of your repository, and add it to version control.
+
+```bash
+git add CITATION.cff
+git commit -m "Add citation file"
+git push
+```
+ 
+## HERMES configuration
+ 
+The HERMES workflow is configured in a [TOML](https://toml.io) file: `hermes.toml`.
+Each step in the publication workflow has its own section.
+
+Configure HERMES to:
+
+- harvest metadata from Git and `CITATION.cff`
+- skip validation of `CITATION.cff`
+- deposit on Zenodo Sandbox (which is built on the InvenioRDM)
+- use Zenodo Sandbox as the target publication repository
+
+```toml
+[harvest]
+from = [ "cff", "git" ]
```

Comment:
I think we really need a decision on the executions order/merge order https://github.com/hermes-hmc/workflow/issues/156 , because with the current implementation this tutorial suggests to prioritize information harvested from git over that manually entered meta data from cff.

If we keep the current ordering, a more sensible suggestion would be:
```toml
form = ["git", "cff"]
```

Reply:
Done in https://github.com/hermes-hmc/workflow/pull/180/commits/e18fadcf74059a30318c4fab4c5e08df0b74ce56

Reply:
I would prefer to simply keep out the `git` for now as it does not provide to much usable output based on uncertain semantics...